app/react/Metadata/components/Metadata.js

```diff
@@ -48,18 +52,18 @@ const removeEmptyValues = (p) => {
 
 const Metadata = ({ metadata, compact, renderLabel }) => (
   <React.Fragment>
-    {metadata.filter(removeEmptyValues).map((prop) => {
-      let type = prop.type ? prop.type : 'default';
-      type = type === 'image' || type === 'media' ? 'multimedia' : type;
-      return (
-        <dl className={`metadata-type-${type} ${prop.fullWidth ? 'full-width' : ''}`} key={prop.label}>
-          {renderLabel(prop, <dt>{t(prop.translateContext, prop.label)}</dt>)}
-          <dd className={prop.sortedBy ? 'item-current-sort' : ''}>
-            {showByType(prop, compact)}
-          </dd>
-        </dl>
-      );
-    })}
+    {metadata.filter(removeEmptyValues).map((prop, index) => {
+          let type = prop.type ? prop.type : 'default';
+          type = type === 'image' || type === 'media' ? 'multimedia' : type;
+          return (
+            <dl className={`metadata-type-${type} ${prop.fullWidth ? 'full-width' : ''}`} key={`${prop.name}_${index}`}>
```

Comment:
The previous comment probably originates here: prop.label has been replaced with prop.name.

If this is correct, the fixtures need to reflect the additional name so they don't show as undefined.
If it is not correct, please use prop.label.